pillar.example

```diff
@@ -118,7 +118,7 @@ nginx:
       # see also http://nginx.org/en/docs/example.html
       #-- - - - - -- - - -- - - - - -- - - -- - - - -- - - - - - -- - - - - - -- - - - - -- - - - - -- - - #
       config:
-        include 'snippets/letsencrypt.conf'
+        include: 'snippets/letsencrypt.conf'
```

Comment:
@garrettw Good catch.  Would you mind applying the same fix for the following two lines as well?

1. https://github.com/saltstack-formulas/nginx-formula/blob/c604d501d65313742d2a6e7eb0e58eb570cdd8bc/pillar.example#L211
1. https://github.com/saltstack-formulas/nginx-formula/blob/c604d501d65313742d2a6e7eb0e58eb570cdd8bc/pillar.example#L224

Reply:
done